gui/wxpython/datacatalog/infomanager.py

```diff
@@ -57,5 +57,18 @@ def ShowImportDataInfo(self, OnImportOgrLayersHandler, OnImportGdalLayersHandler
         ).format(loc=gisenv()['LOCATION_NAME'])
         self.infoBar.ShowMessage(message, wx.ICON_INFORMATION, buttons)
 
+    def ShowImportSuccessfulInfo(self):
+        """Show info about the Display and Modules tabs focused on the first-time user"""
+        buttons = [("See documentation", self._onSeeDocumentation)]
```

Comment:
The reason for the button is unclear to me. The modules are in the Modules tab, so there is no need for the button if that's what it is drawing attention to.